src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -147,6 +152,28 @@
  *   &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example to allow only one and only parameterized annotation on the same line
+ * </p>
+ * <pre>
+ * &#64;SuppressWarnings("deprecation") DataLoader loader; // ok
+ * &#64;Mock DataLoader loader; // violation
+ * </pre>
+ * <p>
+ * Use the following configuration to only validate annotations on methods:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;AnnotationLocation&quot;&gt;
+ *   &lt;property name=&quot;tokens&quot; value=&quot;METHOD_DEF&quot;/&gt;
+ *  &lt;/module&gt;
+ * </pre>
+ * <p>
+ * Example to check default configuration with each method
```

Comment:
Please add a colon `:` after this sentence

Reply:
Done